config/tools-manual.json

```diff
@@ -341,10 +341,11 @@
                 "title": "Bump.sh",
                 "description": "OpenAPI 2 & 3 / AsyncAPI 2 documentation generator, with automatic changelog and visual diff.",
                 "links": {
-                    "websiteUrl": "https://bump.sh/"
+                    "websiteUrl": "https://bump.sh/",
+                    "docsUrl": "https://docs.bump.sh/help/"
                 },
                 "filters": {
-                    "categories": ["documentation-generator"],
+                    "categories": ["documentation-generator", "directory"],
```

Comment:
@philsturgeon can you specify how come it is a directory of AsyncAPI 

till now this category had pages that were aggregating different AsyncAPI documents in one place

Reply:
I'm aware of hubs, but it is a product feature, something that organization can use, not a directory of example AsyncAPI Documents like the ones we already have under https://www.asyncapi.com/tools#Directories

Reply:
If that's only a list of Public Directories then understandable, this would allow Organization Directories. Might be worth clarifying that somewhere.

Reply:
when you go to `Directories` section, there is a paragraph `The following is a list of directories that index public AsyncAPI documents.` so we can always rewrite it to sound better and clearer